release-notes.md

```diff
@@ -32,6 +32,44 @@ When we change the API in a backwards-incompatible way, we release a new minor v
 
 ## Changes
 
+### 15 December 2017
+{: #15-december-2017}
+
+- Dutch [custom model](customizing.html) support is now available for entities and relations.
+- The French concepts and sentiment models have been improved to deliver better results.
+- We switched to a faster language detection model that detects more languages overall. See the complete list of detectable languages [here](detectable-languages.html).
```

Comment:
Don't need to use first person here. "The language detection model now detects more languages faster." 

And would be better to stick to the standard lead-in, "For a complete list..., see [Detectable languages].", to get away from the "click here" kind of link.